Description: Do not #include <linux/autoconf.h> on modern kernels.
 This header is automatically included into every compilation unit by 
 the kernel build system as of kernel 2.6.15, and its location has moved
 from <linux/autoconf.h> to <generated/autoconf.h> in kernel 2.6.33.
Author: Anders Kaseorg <andersk@mit.edu>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/bcmwl/+bug/580594
Last-Update: 2010-05-14

diff -ur src/src/include/linuxver.h src/src/include/linuxver.h
--- src/src/include/linuxver.h	2010-02-05 20:59:15.000000000 -0500
+++ src/src/include/linuxver.h	2010-05-14 21:32:12.869043467 -0400
@@ -19,7 +19,7 @@
 #include <linux/version.h>
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 0))
 #include <linux/config.h>
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 15)
 #include <linux/autoconf.h>
 #endif
 #include <linux/module.h>
